ts/tat-trend.ts

```diff
@@ -597,13 +601,22 @@ function buildTableFromMetrics(
           const daysForTimeRange = groupDays[timeRange];
           const totalDays = daysForTimeRange.reduce((a, b) => a + b, 0);
           const averageDays =
-            totalCases > 0 ? (totalDays / totalCases).toFixed(1) : "0";
+            totalCases > 0
+              ? parseFloat((totalDays / totalCases).toFixed(1))
+              : undefined;
           const medianDays =
-            totalCases > 0 ? calcMedian(daysForTimeRange).toFixed(1) : "0";
+            totalCases > 0
+              ? parseFloat(calcMedian(daysForTimeRange).toFixed(1))
+              : undefined;
+
           assayMetrics.timeRanges.push({
             group: timeRange,
-            avgDays: averageDays,
-            medianDays,
+            avgDays:
+              averageDays !== undefined
+                ? averageDays.toString()
+                : NOT_AVAILABLE,
+            medianDays:
+              medianDays !== undefined ? medianDays.toString() : NOT_AVAILABLE,
```

Comment:
You're setting these values to undefined and then changing undefined to N/A right below. Either store as undefined in the time range (and handle that appropriately elsewhere), or set these to N/A initially.